presto-product-tests/src/main/java/io/prestosql/tests/hive/acid/TestPartitionedFullAcidTable.java

```diff
@@ -0,0 +1,134 @@
+import io.prestosql.tempto.Requirement;
+import io.prestosql.tempto.RequirementsProvider;
+import io.prestosql.tempto.configuration.Configuration;
+import io.prestosql.tempto.context.ThreadLocalTestContextHolder;
+import io.prestosql.tempto.fulfillment.table.MutableTablesState;
+import io.prestosql.tempto.fulfillment.table.hive.HiveTableDefinition;
+import io.prestosql.tempto.query.QueryExecutor;
+import io.prestosql.tempto.query.QueryResult;
+import org.apache.thrift.TException;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+
+import static io.prestosql.tempto.Requirements.compose;
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.fulfillment.table.TableRequirements.mutableTable;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.HIVE_ACID;
+import static io.prestosql.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.prestosql.tests.hive.acid.AcidTestHelper.simulateAbortedHiveTranscation;
+
+public class TestPartitionedFullAcidTable
+        extends ProductTest
+        implements RequirementsProvider
+{
+    private static String tableName = "ORC_acid_single_int_column_partitioned_fullacid";
+
+    private static final HiveTableDefinition SINGLE_INT_COLUMN_PARTITIONED_ORC = singleIntColumnPartitionedTableDefinition(Optional.empty());
+
+    private static HiveTableDefinition singleIntColumnPartitionedTableDefinition(Optional<String> serde)
+    {
+        return HiveTableDefinition.builder(tableName)
+                .setCreateTableDDLTemplate(buildSingleIntColumnPartitionedTableDDL(serde))
+                .setNoData()
+                .build();
+    }
+
+    private static String buildSingleIntColumnPartitionedTableDDL(Optional<String> rowFormat)
+    {
+        StringBuilder sb = new StringBuilder();
+        sb.append("CREATE %EXTERNAL% TABLE IF NOT EXISTS %NAME%(");
+        sb.append("   col INT,");
+        sb.append("   fcol INT");
+        sb.append(") ");
+        sb.append("PARTITIONED BY (part_col INT) ");
+        if (rowFormat.isPresent()) {
+            sb.append("ROW FORMAT ").append(rowFormat.get());
+        }
+        sb.append(" STORED AS ORC");
+        sb.append(" TBLPROPERTIES ('transactional'='true')");
+        return sb.toString();
+    }
+
+    @Inject
+    private MutableTablesState tablesState;
+
+    @Override
+    public Requirement getRequirements(Configuration configuration)
+    {
+        return compose(mutableTable(SINGLE_INT_COLUMN_PARTITIONED_ORC));
+    }
+
+    @Test(groups = {HIVE_ACID, PROFILE_SPECIFIC_TESTS})
+    public void testReadingPartitionedInsertOnlyAcidTable()
+    {
+        String tableNameInDatabase = tablesState.get(tableName).getNameInDatabase();
+
+        QueryExecutor hiveQueryExecutor = ThreadLocalTestContextHolder.testContext().getDependency(QueryExecutor.class, "hive");
```

Comment:
Same comment as in https://github.com/prestosql/presto/pull/1034 -- use `onHive()....`